Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/GitHub4720.cs

```diff
@@ -49,6 +49,11 @@ public async Task GitHub4720Test()
 
 #endif
 	}
+	
+	public class _4720WebView : WebView
```

Comment:
Can you move these two classes into `class GitHub4720`? It helps keeps the `Xamarin.Forms.Controls.Issues` namespace a bit cleaner